tests/lib/reset.sh

```diff
@@ -72,8 +70,15 @@ reset_classic() {
         # Restore snapd state and start systemd service units
         restore_snapd_state
         escaped_snap_mount_dir="$(systemd-escape --path "$SNAP_MOUNT_DIR")"
-        mounts="$(systemctl list-unit-files --full | grep "^${escaped_snap_mount_dir}[-.].*\\.mount" | cut -f1 -d ' ')"
-        services="$(systemctl list-unit-files --full | grep "^${escaped_snap_mount_dir}[-.].*\\.service" | cut -f1 -d ' ')"
+        all_units="$(systemctl list-unit-files --full | cut -f1 -d ' ')"
+        mounts=""
+        if echo "$all_units" | grep "^${escaped_snap_mount_dir}[-.].*\\.mount"; then
```

Comment:
Small suggestion, when using grep this way, pass -q to make the it silent:
```suggestion
        if echo "$all_units" | grep -q "^${escaped_snap_mount_dir}[-.].*\\.mount"; then
```

Reply:
done